F/FastJet_Julia_Wrapper/build_tarballs.jl

```diff
@@ -3,10 +3,9 @@
 using BinaryBuilder
 using Pkg
 
-julia_version = v"1.5.3"
-
 name = "FastJet_Julia_Wrapper"
-version = v"0.8.6"
+version = v"0.9.0"
+julia_versions = [v"1.6", v"1.7"]
```

Comment:
Unless it fails to build with 1.8, please also add that, so that people can also test with Julia nightly... Although I guess we should also update the Julia 1.8 (and 1.7!) binaries.
```suggestion
julia_versions = [v"1.6", v"1.7", v"1.8"]
```